### DISCUSSION TITLE
What is a turophile?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#741] [Galaxy Trivia] [answered]
*answer with no words* 🙃

**Comment by SANKARAMDAS** [comment]
🧀😍 

![download](https://github.com/ParanoidUser/yolo/assets/31897843/1f206d72-489b-4766-b161-eb2460a7d004)

**Comment by peterbaccaro** [comment, marked as answer]
🧀 ❤️ 😍

![cheese](https://github.com/ParanoidUser/yolo/assets/74666079/e83ea964-3648-4274-a702-09eba6a20cfc)

**Comment by ParanoidUser** [comment]
@carlostipton1319, do you want to take a shot at this one?
